packages/frontend/web/browser/startup.ts

```diff
@@ -13,13 +13,21 @@ const measure = (name: string, task: () => Promise<void>): void => {
 
     perf.mark(start);
 
-    task().finally(() => {
+    function markEnd() {
         perf.mark(end);
         perf.measure(name, start, end);
 
         // tslint:disable-next-line:no-console
         console.log(JSON.stringify(perf.getEntriesByName(name)));
-    });
+    }
+
+    task()
+        // You could use 'finally' here to prevent this duplication but finally isn't supported
```

Comment:
Does this mean we are not retro-fitting `finally` with polyfill.io or babel? And if so, should we fix that issue? e.g. https://polyfill.io/v3/polyfill.min.js?features=Promise.prototype.finally as presumably if it is broken in Cypress it is broken in real browsers too?

Reply:
`Finally` is included in the polyfill.io response for Chrome 59 so this should work...:

![image](https://user-images.githubusercontent.com/638051/66635941-17850d80-ec08-11e9-8808-e071998c9fab.png)

Reply:
I guess we're hitting this issue (detailed [here](https://github.com/cypress-io/cypress/issues/2651#issuecomment-432698837)) because the server is running in dev mode, so not transpilled.

We probably want to be executing cypress against a prod version of the app. To do this would need the `make cypress` command updating. I had some issues doing that though (I had to manually execute `sudo mkdir /var/logs/dotcom-rendering` to make it work) so I thought it better to hold off that approach for now.

Reply:
> I guess we're hitting this issue (detailed here) because the server is running in dev mode, so not transpilled.

Hmm, if the polyfill is coming from `polyfill.io` though it's not due to transpilation. Polyfill.io should always run before anything else (either mustard is cut or queue the measuring) - see https://github.com/guardian/dotcom-rendering/pull/756/files so this is odd - maybe the mustard is being cut without supporting `Finally` for some reason?

Reply:
Ah, I see - so polyfill _should_ be running and _should_ be transpilling the code. That is odd. It does sound like polyfill isn't handling `finally`... Tempting to just not use `finally` rather than wade through [the internet](https://www.google.com/search?q=polyfill+finally) but that's a poor solution for future developers so I'm happy to hold off merging this in favour of some investigation.